src/System.Management.Automation/engine/TypeTable_Types_Ps1Xml.cs

```diff
@@ -6058,7 +6058,7 @@ private void Process_Types_Ps1Xml(string filePath, ConcurrentBag<string> errors)
           {
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Get-PSCallStack DRT Unit Tests.Verify that the script block of a trap statement shows up on the call stack
```powershell
Expected 77, but got 80.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-PSCallStack.Tests.ps1: line 85
85:         $results[0].InvocationInfo.ScriptLineNumber | Should -Be 77
```